runtime/doc/eval.txt

```diff
@@ -5506,6 +5506,66 @@ max({expr})	Return the maximum value of all items in {expr}.
 		items in {expr} cannot be used as a Number this results in
                 an error.  An empty |List| or |Dictionary| results in zero.
 
+menu_get({path}, {modes})				*menu_get()*
+		Returns a |Dictionary| with all the submenu of {path} (set to 
+		an empty string to match all menus). Only the commands matching {modes} are 
+		returned ('a' for all, 'i' for insert see |creating-menus|).
+
+    For instance, executing:
+<
+    nnoremenu &Test.Test inormal
+    inoremenu Test.Test insert
+    vnoremenu Test.Test x
+    echo menu_get("")
+>
+should produce an output with a similar structure:
+<
```

Comment:
several things were wrong with the doc ('>' pairs, spaces vs tabs, json format etc). I updated it and checked, I think it's fine now.